src/components/nav.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&family=Roboto+Slab:wght@200;400;600&display=swap");
+
+.nav1 {
+  color: #0290ff;
+  width: 15rem;
```

Comment:
- [ ] Let's try to make the cards more similar to the ones on the Zeplin template. I think that with these kinds of projects, ideally, we want to match 90 ~ 100% of the design. To do this, we can implement the missing components, like the progress bar, the vertical divider, and the current chapter. (Even if they are just placeholders for now)

For reference this is an image of your book cards:
![image](https://user-images.githubusercontent.com/79114233/146472331-2ddb9f4c-fe2a-41d7-a89b-57c54da08893.png)

And this is from the template: 
![image](https://user-images.githubusercontent.com/79114233/146472358-9432a81f-4dc1-4209-847d-c5a7f51010a0.png)

- While we are it, I think we can also fix the title, so if it occupies two lines the author does not get under it. Let's also remove the borders from the default borders from the buttons, I think that will make the page look much cleaner.


[OPTIONAL] I recommend avoiding fixing the width of certain components, especially those that are too big. Instead, we can use more flexible options, like max-width or min-width, and responsive-friendly units like: `vh` and `%`.